### PR TITLE
Add some two more Trenz boards, one containing Virtex7

### PR DESCRIPTION
**Pull request by UweBonnes**

### CODE REVIEW

File: doc/boards.yml [hunk 1]
```diff
@@ -735,6 +735,12 @@
   FPGA: Gowin Arora V GW5AST-138B (GW5AST-LV138FPG676A)
   Memory: OK
   Flash: TBD
+- ID: te0712_8
+  Description: Trenz Electronic TE0712 FPGA-Module mit AMD Artix™ 7(TE0712)
+  URL: https://shop.trenz-electronic.de/en/TEC0117-01-FPGA-Module-with-GOWIN-LittleBee-and-8-MByte-internal-SDRAM
+  FPGA: XC7A200TFBG484
+  Memory: OK
+  Flash: IF
```
Comment (trabucayre) [inline doc/boards.yml:743]: Copy/paste from Trenz littlebee board? URL seems not true and Flash must be *OK* (IF is for internal flash)

File: doc/boards.yml [hunk 2]
```diff
@@ -750,6 +756,13 @@
   Memory: OK
   Flash: NT
 
+- ID: tec0330
+  Description: PCIe FMC Carrier with Xilinx Virtex-7 FPGA (TEC0330)
+  URL: https://shop.trenz-electronic.de//TEC0330-05-PCIe-FMC-Carrier-with-Xilinx-Virtex-7-FPGA-8-Lane-PCIe-GEN2-SODIMM-SDRAM
+  FPGA: XC7VX330T-2FFG1157C
+  Memory: OK
+  Flash: IF
```
Comment (trabucayre) [inline doc/boards.yml:764]: must be OK instead of IF

**Comment by trabucayre** [comment]
LGTM. Thanks!
Just a small fix is required.

File: doc/boards.yml (hunk 2 above)
Comment (UweBonnes) [reply doc/boards.yml]: Fixed with first rebase.

File: doc/boards.yml [hunk 3]
```diff
@@ -735,6 +735,12 @@
   FPGA: Gowin Arora V GW5AST-138B (GW5AST-LV138FPG676A)
   Memory: OK
   Flash: TBD
+- ID: te0712_8
+  Description: Trenz Electronic TE0712 FPGA-Module mit AMD Artix™ 7(TE0712)
+  URL: https://shop.trenz-electronic.de/en/TEC0117-01-FPGA-Module-with-GOWIN-LittleBee-and-8-MByte-internal-SDRAM
```
Comment (trabucayre) [inline doc/boards.yml:740]: This is the URL for the TEC0117 instead of TE0712.

**Comment by trabucayre** [comment]
Applied Thanks @UweBonnes

**Pull request merged**